neore/scripts/package.sh

```diff
@@ -67,6 +67,10 @@ function download_flume(){
     rm -f ${FLUME_WO_TAR}/lib/libthrift-*.jar
     mv ${LIBTHRIFT_JAR} ${FLUME_WO_TAR}/lib
 
+    _log "### Disable httpclient and httpcore libraries distributed within apache-flume bundle... ###"
+    mv ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar.old
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/fbff74f503a3d5ddf5c0347842f97a521d31e812